pkg/services/cloudmigration/cloudmigrationimpl/xorm_store_test.go

```diff
@@ -5,7 +5,6 @@ import (
 	"strconv"
 	"testing"
 
-	"github.com/bmizerany/assert"
```

Comment:
Note: this was adding a new dependency to `go.mod` with this unmaintained package.